packages/docusaurus-theme-classic/src/theme/DocItem/index.js

```diff
@@ -79,7 +79,11 @@ function DocItem(props) {
   return (
     <>
       <Head>
-        {title && <title>{title}</title>}
+        {title && (
+          <title>
+            {title} · {siteTitle}
```

Comment:
I think using `-` or `|` will be better

Reply:
@yangshun done, but what about avoiding to use tagline if there is no title? I mentioned this in the PR description.

`${siteTitle} · ${tagline}` -> `${siteTitle}`

Instead, use tagline only on the home page.

Reply:
Agreed. Don't use tagline elsewhere. I think if the tagline is too long it even looks weird to have it as the <title> on the homepage, just like ours :S

Reply:
Excluded tagline from meta title of any page, but bring back it to title on home page.